lib/redmine_tags/patches/wiki_controller_patch.rb

```diff
@@ -1,3 +1,21 @@
+# This file is a part of redmine_tags
```

Comment:
I'm OK to remove if from headings of all files as well as simply skip it for new ones only...
In fact I was just following [GPL's How To](https://www.gnu.org/licenses/gpl-howto.html).
And I saw this practice in some other products... But I am absolutely agree that updating
year each time will be creating stupid huge commit... Pretty useless one.

After all, again, I'm absolutely fine to remove that heading from each file.
LICENSE alone is more than enough I believe.

Also, in fact Copyright line became pretty outdated ;))
Probably we should change it to:

```
Copyright (c) 2010 Alexey Zapparov
Copyright (c) 2011-2016 Redmine Tags contributors
```

And generate CONTRIBUTORS file upon releases.